### PR TITLE
Add capability for TOMAS simulations in GCHP

### PR DESCRIPTION
**Pull request by msulprizio**
### Name and Institution (Required)

Name: Melissa Sulprizio
Institution: Harvard / GCST

### Describe the update

This pull request adds the option to run GCHP with TOMAS as originally implemented by @BettyCroft in:

- https://github.com/geoschem/GCHP/pull/252

This pull request should be merged alongside:
- https://github.com/geoschem/geos-chem/pull/2060
- https://github.com/geoschem/HEMCO/pull/251

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +40,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Fixed
 - Fixed post-advection pressure edges (PLE1) passed to advection to be derived from the correct surface pressure
 - Fixed typo in `docs/source/conf.py`, "_static" should be "_static/"
+>>>>>>> origin/main
```
Comment (yantosca) [inline CHANGELOG.md]: Delete leftover slug from the git merge

**Review by yantosca — CHANGES_REQUESTED** [review]
This looks mostly good to merge, pending minor fixes as noted.

Also: The HEMCO submodule only has 2 files changed but the HEMCO PR #251 contains changes 3 files.  We'll need to point to the updated commit in the HEMCO submodule.

File: CHANGELOG.md (hunk 1 above)
Comment (msulprizio) [reply CHANGELOG.md]: This is now fixed in https://github.com/geoschem/GCHP/pull/359/commits/41c136fc1e725e98a54216d7df2bcdb94f7fbaa3.

**Review by yantosca — APPROVED** [review]
Now good to merge!

**Pull request merged**